client/src/components/AuthHeader/useStyles.ts

```diff
@@ -1,17 +1,23 @@
 import { makeStyles } from '@material-ui/core/styles';
+import { custom } from '../../themes/custom';
 
 const useStyles = makeStyles((theme) => ({
   authHeader: {
+    height: '100%',
     alignSelf: 'flex-end',
```

Comment:
Most of these styles can be applied to the Box component :)

Eg: `<Box display="flex" alignItems="center"....>`

Reply:
Done.